docs/LoaderInterfaceArchitecture.md

```diff
@@ -640,7 +667,8 @@ discovery.
         continue to work.
     </small></td>
     <td><small>
-        If a relative path not used, issues may be encountered.
+        If a relative path not used, issues may be encountered due to dynamic
+        library lookup
```

Comment:
This is a nitpick comment. Feel free to ignore, or fix.

Reply:
Its a good comment, turns out that text is a mistake introduced during a rebase of a previous PR.

I went ahead and created #1554 to clean it up. The note now includes "why" its a good idea to use an absolute path, rather than a cryptic "you shouldn't use relative paths" for no obvious reason.